Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_AWS_IAM_user_access_investigation.yml

```diff
@@ -1364,6 +1374,106 @@ tasks:
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
+  "44":
+    id: "44"
+    taskid: 26a9e1e8-787c-4528-8a78-b164a059d66b
+    type: regular
+    task:
+      id: 26a9e1e8-787c-4528-8a78-b164a059d66b
+      version: -1
+      name: AWS IAM - Get user profile
+      description: Retrieves the user login profile information for AWS management
+        consle
+      script: '|||aws-iam-get-user-login-profile'
```

Comment:
we should add to the task "skip if automation / integration" unavailable check to true